TS3AudioBot/NLog.config

```diff
@@ -3,25 +3,34 @@
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       throwConfigExceptions="true">
 
-	<targets>
-		<!-- Config log to file -->
-		<target layout="${longdate}|${pad:padding=5:inner=${level:uppercase=true}}|${callsite:includeNamespace=false} ${message}${onexception:${newline}${exception:format=tostring}}" name="logfile" xsi:type="File" fileName="ts3audiobot.log" />
-		<!-- Config log to console -->
-		<target xsi:type="ColoredConsole"
+  <targets>
+    <!-- Config log to file -->
+    <target layout="${longdate}|${pad:padding=5:inner=${level:uppercase=true}}|${callsite:includeNamespace=false} ${message}${onexception:${newline}${exception:format=tostring}}" name="logfile" xsi:type="File" fileName="/var/log/ts3ab/bot.log" />
```

Comment:
Please undo this, not everyone runs this on a linux server.

Reply:
my intention was not for you to push this but for me to have a reference on how i did it even when i delete my fork ^^